client/components/blogging-prompt-card/index.jsx

```diff
@@ -101,7 +102,7 @@ const BloggingPromptCard = ( { siteId, viewContext, showMenu, index } ) => {
 
 	return (
 		<div className="blogging-prompt">
-			<Card className={ classnames( 'customer-home__card', 'blogging-prompt__card' ) }>
+			<Card className={ `blogging-prompt__card ${ isHome }` }>
```

Comment:
nit: I think the name isn't quite right, maybe `homeClassName` or something because isHome makes it sound like a bool

Reply:
I'll update it to use classnames with some logic